src/routing/Router.svelte

```diff
@@ -1,16 +1,3 @@
-<script context="module">
-  // We could grab the value of currentQuery here rather than requiring that it
-  // be passed as an argument, but that would not cause any links that use this
-  // function to update reactively.
-  export function getPath(path, query) {
-    return query ? `${path}?${query}` : path;
-  }
-
-  export function navigate(...args) {
-    page.show(getPath(...args));
-  }
-</script>
-
```

Comment:
For this bugfix, we ideally needed a way to call `navigate` without specifying the path and query string separately. Rather than refactoring `navigate` and `getPath` to reinvent the wheel, I decided to just get rid of these helper functions altogether and use `page.show` directly where needed.

Reply:
Ah this answers my first question.